src/pages/billing/List.js

```diff
@@ -44,7 +44,7 @@ const styles = {
 
 const ProjectListItem = ({ project, project: { roles, status }, isActive }) => {
   const selectableProject = ({ projectName }, isActive) => h(Clickable, {
-    style: { ...styles.projectListItem(isActive), color: isActive ? colors.dark() : colors.accent() },
+    style: { ...styles.projectListItem(isActive), color: isActive ? colors.accent(1.1) : colors.accent() },
```

Comment:
Using `colors.accent()` for both did not meet contrast requirements, so I made the selected case slightly darker rather than changing the color for everything

Reply:
I checked out the branch and made sure that BioDataCatalyist and eLwazi also work with this. Next time, please makes sure that the PR deployment is triggered.